lib/interface/cli/commands/agent/get.cmd.js

```diff
@@ -38,6 +40,11 @@ const command = new Command({
             });
         }
         Output.print(_.map(agents, Agent.fromResponse));
+        const pipelineLink = await getTestPipelineLink(INSTALLATION_DEFAULTS.DEMO_PIPELINE_NAME, false);
+        if (pipelineLink) {
+            console.log(`\nTest pipeline with the name:${colors.cyan(INSTALLATION_DEFAULTS.DEMO_PIPELINE_NAME)} exists for this account.` +
```

Comment:
\nTest pipeline with the name: "${colors.cyan(INSTALLATION_DEFAULTS.DEMO_PIPELINE_NAME)}"

Reply:
added space after column and quotes around pipeline name